bls12_377/src/curves/g1_swu_iso.rs

```diff
@@ -0,0 +1,107 @@
+use ark_ec::{
+    hashing::curve_maps::{swu::SWUConfig, wb::IsogenyMap},
+    models::{
+        short_weierstrass::{Affine, SWCurveConfig},
+        CurveConfig,
+    },
+};
+
+use ark_ff::MontFp;
+
+use crate::{g1, Fq, Fr};
+
+type G1Affine = Affine<SwuIsoParameters>;
+
+#[derive(Clone, Default, PartialEq, Eq)]
+pub struct SwuIsoParameters;
```

Comment:
Let's rename these all into `Config`, as per #136 

```suggestion
pub struct SwuIsoConfig;
```

Reply:
Resolved in #7d2ef2a